code/modules/clothing/suits/armor_suits.dm

```diff
@@ -273,13 +273,13 @@
 
 /obj/item/clothing/suit/armor/riot/sec
 	name = "security riot suit"
-	desc = "A suit of armor with heavy padding to protect against melee attacks. Looks like it might impair movement. This one has security markings on it."
+	desc = "A full-body suit of riot gear that provides excellent protection against blunt force trauma, but little protection otherwise. This suit is covered in red Security stripes."
```

Comment:
```suggestion
	desc = "A full-body suit of riot gear that provides excellent protection against blunt force trauma, fire, and corrosive substances, but little protection otherwise. This suit is covered in red Security stripes."
```